.github/workflows/Linter_Check.yml

```diff
@@ -3,18 +3,25 @@
 
 name: Super-Linter
 
-on: push
+on: [push, pull_request]
 
 jobs:
-  super-lint:
-    name: Lint code base
+  build:
+    name: Lint Code Base
     runs-on: ubuntu-latest
+
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
 
-      - name: Run Super-Linter
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Lint Code Base
         uses: github/super-linter@v4
         env:
```

Comment:
You only just need:
```
VALIDATE_ALL_CODEBASE: true
VALIDATE_JAVASCRIPT_STANDARD: false
```

What's present doesn't work noted by:
![image](https://user-images.githubusercontent.com/83375816/232602601-2ab84fb2-553c-444b-ae8a-bd22e309043d.png)

In addition, you disabled using `ESLint` instead of `standard js`.